Firestore/core/test/firebase/firestore/testutil/testutil.h

```diff
@@ -40,6 +47,25 @@ inline model::ResourcePath Resource(absl::string_view field) {
   return model::ResourcePath::FromString(field);
 }
 
+// Version is epoch time in microseconds. This helper is defined so to match
```

Comment:
"Version" here is ambiguous (it's not clear if you mean the function or the param.

Better to define this just in terms of what it does:

```c++
// Creates a snapshot version from the given version timestamp
//
// @param version a timestamp in microseconds since the epoch

Reply:
done